src/liballoc/boxed.rs

```diff
@@ -1,6 +1,6 @@
 //! A pointer type for heap allocation.
 //!
-//! `Box<T>`, casually referred to as a 'box', provides the simplest form of
+//! [`Box<T>`][Box], casually referred to as a 'box', provides the simplest form of
```

Comment:
Is this link right? Below it is:
```
//! [`Box`]: struct.Box.html 
```

Reply:
I wanted to test if the auto linkage was working but apparently it's not. :'(